mne/cuda.py

```diff
@@ -176,7 +216,7 @@ def _setup_cuda_fft_multiply_repeated(n_jobs, h, n_fft, kind="FFT FIR filtering"
 
             try:
                 # do the IFFT normalization now so we don't have to later
-                h_fft = cupy.array(cuda_dict["h_fft"])
+                h_fft = cupy.asarray(_share_cuda_mem(cuda_dict["h_fft"], n_jobs))
```

Comment:
I think this one should be "cuda" rather than `n_jobs` since we are in the using-CUDA path here, and `n_jobs=1` has been set above
```suggestion
                h_fft = cupy.asarray(_share_cuda_mem(cuda_dict["h_fft"], "cuda"))
```

Reply:
sorry I'm not paying enough attention to catch all these. 

I think I'm distracted by `n_jobs`.  If "cuda" is always passed into `_shared_cuda_mem` then do we need `n_jobs` as a param?

Reply:
Nope I think you can get rid of it

Reply:
done